python/cugraph/cugraph/dask/sampling/uniform_neighbor_sample.py

```diff
@@ -141,10 +141,12 @@ def _mg_call_plc_uniform_neighbor_sample(
         )
         for i, w in enumerate(Comms.get_workers())
     ]
-
-    ddf = dask_cudf.from_delayed(
-        result, meta=create_empty_df(indices_t, weight_t), verify_meta=False
-    ).persist()
+    empty_df = (
+        create_empty_df_with_edge_props(indices_t, weight_t)
+        if with_edge_properties
+        else create_empty_df(indices_t, weight_t)
+    )
+    ddf = dask_cudf.from_delayed(result, meta=empty_df, verify_meta=False).persist()
```

Comment:
Will be over-writted by : https://github.com/rapidsai/cugraph/pull/3170 so please ignore.